2023-08-07-report.org

```diff
@@ -0,0 +1,118 @@
+#+title:  Weekly report for 31th of July 2023 to 7th of August 2023
+#+author: Pierre-Emmanuel Patry
+#+date:   2023-08-07
+
+** Overview
+
+Thanks again to [[https://opensrcsec.com/][Open Source Security, inc]] and [[https://www.embecosm.com/][Embecosm]] for their ongoing support for this project.
+
+*** Milestone Progress
+
+1. Name resolution rework
+  1. Pushed many improvements over the existing code
+  2. Early
+    1. Most of early name resolution has been merged
+    3. Procedural macro name resolution are almost finished
```

Comment:
Missing the 2. here, or most probably, 3 should be 2?

Reply:
Yeah good catch. Do you see anything we should add to this section btw ? Maybe a line for your commits ?

Reply:
My changes are not really part of any milestone, so not really sure.
Maybe we can add "Minor refactor of the macro transcriber functions to use the newly added tl::optional<>".
But I'll leave it to you to decide if that's worth a mention.